indicators/bank-of-america-kgzrkd.yml

```diff
@@ -0,0 +1,31 @@
+title: Bank of America Phishing Kit kgzRkD
+description: |
+    Detects a phishing kit targeting Bank of America. This kit is already detected by Urlscan.
+    Found as a result of it being deployed on Replit.
+
+
+references:
+    - https://urlscan.io/result/928c09f2-1a08-4a13-a76e-4c8c5e741063/
```

Comment:
Might want to add another reference here:

`https://urlscan.io/result/3377f34c-fa5a-4171-8ea4-6501c3b70c9b`